### PR TITLE
clone is created

### PR DESCRIPTION
**Pull request by PRABgit01**
Screenshot 2024-01-03 at 1 35 10 AM
Screenshot 2024-01-03 at 2 34 45 PM

Screenshot 2024-01-03 at 2 35 01 PM
first section
Screenshot 2024-01-03 at 2 35 34 PM
second section 
Screenshot 2024-01-03 at 2 35 50 PM
scrollable feature
Screenshot 2024-01-03 at 2 36 06 PM

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Chainpal code</title>
```
Comment (ankitadrixus08) [inline Index.html]: Chainpal code >>> Chainpals

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        <link href="./styled.css" rel="stylesheet">
+
+    </head>
+    <body>
+        <div class="wrapper">
+            <section class="bannerSection">
+                <div class="container">
+                    <h3 class="h3CustomMobile">
```
Comment (ankitadrixus08) [inline Index.html]: h3CustomMobile >> heading, title with another class of mobileVersion

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+                        Join the Escrow Manager Program
+                    </h3>
+                    <div class="innerWrapper">
+
+                        <div class="leftDiv">
+                            <div class="iamgeDiv">
```
Comment (ankitadrixus08) [inline Index.html]: iamgeDiv > > imageDiv

Comment (ankitadrixus08) [reply Index.html]: also name should be imageBox

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+                                <img class="imageDesign"
```
Comment (ankitadrixus08) [inline Index.html:19]: need alt tag @PRABgit01

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+                                    src="https://www.chainpals.io/assets/images/guest/png/become-escrow-manager.png">
+                            </div>
+
+                        </div>
+                        <div class="rightDiv">
+                            <div class="rightDivContainer">
+                                <h3 class="h3Custom">
```
Comment (ankitadrixus08) [inline Index.html]: h3Custom >> heading, title @PRABgit01

File: Index.html [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+                                    Join the Escrow Manager Program
+                                </h3>
+                                <p class="description">
+                                    Become a Chainpals Escrow Manager and start
+                                    earning passive income by hodling EM NFT.
+                                </p>
+
+                                <div class="button">
+                                    <a>LEARN MORE</a>
```
Comment (ankitadrixus08) [inline Index.html]: it should be clickable link @PRABgit01

Comment (ankitadrixus08) [reply Index.html]: also href with this @PRABgit01

File: Index.html [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+                </section class="roadMapSection">
```
Comment (ankitadrixus08) [inline Index.html]: why this @PRABgit01

File: Index.html [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+                <!-- --------next section---------- -->
+                <section>
```
Comment (ankitadrixus08) [inline Index.html]: need class here @PRABgit01

File: Index.html [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+                                    <a>LEARN MORE</a>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+                </section class="roadMapSection">
+                <!-- --------next section---------- -->
```
Comment (ankitadrixus08) [inline Index.html]: make proper comments like roadmap section start @PRABgit01

File: Index.html [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+                <section>
+                    <div class="roadMApContainer">
+                        <p class="roadMapTitle">ROAD MAP</p>
+                    </div>
+                    <div class="backgroundImagefeature">
+                        <img src="https://www.chainpals.io/assets/images/guest/svgs/roadmap.svg" class="secondimg">
```
Comment (ankitadrixus08) [inline Index.html]: secondimg >> why second image ????? please give proper naming @PRABgit01

Comment (ankitadrixus08) [reply Index.html]: use alt tag in every images @PRABgit01

File: Index.html [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+                    </div>
+                    <div class="roadMApWrapper">
```
Comment (ankitadrixus08) [inline Index.html]: roadMApWrapper >> roadMapWrapper

File: Index.html [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+                        <div class="swiperWrapper">
+                            <div class="swiperWrapperText">
+                                <div class="timeStampText">
+                                    <span class="date">Q4 2021</span>
```
Comment (ankitadrixus08) [inline Index.html]: date >> roadmapDate 

please don't use preserved variable in class name @PRABgit01

File: Index.html [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+                                </div>
+                                <div class="statusWrapper">
+
```
Comment (ankitadrixus08) [inline Index.html]: remove unwanted space @PRABgit01

File: Index.html [hunk 14]
```diff
@@ -0,0 +1,215 @@
...
+                                    <div class="status">
+                                        <div class="timeStampText">
+                                            <span class="dateMobile">Q4 2021</span>
```
Comment (ankitadrixus08) [inline Index.html]: dateMobile >> roadmapDate mobile

@PRABgit01

File: Index.html [hunk 15]
```diff
@@ -0,0 +1,215 @@
...
+                                        </div>
+                                        <p class="description"> Brainstorm of
+                                            website, </p>
+                                        <p _ngcontent-jil-c115
```
Comment (ankitadrixus08) [inline Index.html]: _ngcontent-jil-c115 why this ?

File: Index.html [hunk 16]
```diff
@@ -0,0 +1,215 @@
...
+                                            class="description"> Initiate
+                                            wireframes technical documents,
+                                            create whitepaper & tokenomics. </p>
+
+                                    </div>
+                                </div>
+                            </div>
+
+                            <!-- -----------------nextsection -->
```
Comment (ankitadrixus08) [inline Index.html]: give proper comments @PRABgit01

File: Index.html [hunk 17]
```diff
@@ -0,0 +1,215 @@
...
+                                        </div>
+                                        <p class="description">  Testing & go-live preparations for pre-sale of tokens.  </p>
+                                        <p _ngcontent-jil-c115
+                                            class="description">  Launch of pre-sale of the tokens.  </p>
+
+                                    </div>
+                                </div>
+                            </div>
+                            <!-- -----------------nextsection -->
+                            <!-- -----------------nextsection -->
```
Comment (ankitadrixus08) [inline Index.html]: why this same comments multiple time @PRABgit01

Comment (ankitadrixus08) [reply Index.html]: remove unwanted space and code from document @PRABgit01

File: styled.css [hunk 18]
```diff
@@ -0,0 +1,308 @@
...
+  display: flex;
+  justify-content: center;
+}
+.imageDesign {
+  max-width: 342px;
+  width: 100%;
+}
+.description {
+  color: #fff;
+  font-size: 16px;
```
Comment (ankitadrixus08) [inline styled.css:40]: don't need line-height here? @PRABgit01

File: styled.css [hunk 19]
```diff
@@ -0,0 +1,308 @@
...
+}
+
+.button {
+  background-color: #0d4df2;
+  border: 1.5px solid #0d4df2;
```
Comment (ankitadrixus08) [inline styled.css]: make 1px or 2px here @PRABgit01

File: styled.css [hunk 20]
```diff
@@ -0,0 +1,308 @@
...
+  height: 16px;
+  font-size: 14px;
+  padding: 15px 30px;
+  color: #fff;
+  border-radius: 8px;
+  width: 20%;
+  margin: 1px;
+  width: 20%;
```
Comment (ankitadrixus08) [inline styled.css]: why multiple time @PRABgit01 remove here

File: styled.css [hunk 21]
```diff
@@ -0,0 +1,308 @@
...
+}
+
+.h3CustomMobile {
+  visibility: hidden;
+}
+
+/* -------CSSFOR DIFFERENT SECTION-------- */
+.inerWrapperRoadMap {
+  display: flex;
+  justify-content: center;
```
Comment (ankitadrixus08) [inline styled.css]: you don't required align-item here @PRABgit01

File: styled.css [hunk 22]
```diff
@@ -0,0 +1,308 @@
...
+}
+
+.roadMApContainer {
```
Comment (ankitadrixus08) [inline styled.css]: why you make same container here @PRABgit01  you can use same container for whole website not different for every section

File: styled.css [hunk 23]
```diff
@@ -0,0 +1,308 @@
...
+
+.roadMapTitle {
+  margin: 64px;
+  color: #c6cdd9;
+  text-align: center;
+  font: sans-serif;
+  font-size: 48px;
+}
+.roadMApWrapper {
+  margin-top: 40px;
```
Comment (ankitadrixus08) [inline styled.css:91]: remove space @PRABgit01

File: styled.css [hunk 24]
```diff
@@ -0,0 +1,308 @@
...
+  width: 16px;
+  height: 16px;
+  background-color: #fff;
+  border-radius: 25px;
+  top: -35px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+}
+.description {
```
Comment (ankitadrixus08) [inline styled.css]: remove this @PRABgit01

File: styled.css [hunk 25]
```diff
@@ -0,0 +1,308 @@
...
+}
+
+.backgroundImagefeature {
+ position: absolute;
+    background-size: contain;
+    max-width: 390px;
+    width: 100%;
+    height: 390px;
+    left: 0;
+    top: 473px;
```
Comment (ankitadrixus08) [inline styled.css]: why top @PRABgit01

File: styled.css [hunk 26]
```diff
@@ -0,0 +1,308 @@
...
+    z-index: -1;
+}
+
+.secondimg {
+  width: 319px;
```
Comment (ankitadrixus08) [inline styled.css:177]: why give fix width here @PRABgit01

File: styled.css [hunk 27]
```diff
@@ -0,0 +1,308 @@
...
+  text-align: center;
+
+}
+.dateMobile{
+    display: none;
+}
+@media (max-width: 991px) {
+  .h3Custom {
+    visibility: hidden;
+    font-size: 0;
```
Comment (ankitadrixus08) [inline styled.css]: what you mean by font-size 0? @PRABgit01

**Review by ankitadrixus08 — COMMENTED** [review]
please do proper practice in given time 

- do proper reading of html / css
- read naming documentation
- remove unwanted commented code
- remove white space
- give proper indent 


25m

File: Index.html (hunk 1 above)
Comment (PRABgit01) [reply Index.html]: fixed this

File: Index.html (hunk 6 above)
Comment (ankitadrixus08) [reply Index.html]: pending @PRABgit01

File: styled.css (hunk 21 above)
Comment (ankitadrixus08) [reply styled.css]: pending @PRABgit01

File: styled.css (hunk 26 above)
Comment (ankitadrixus08) [reply styled.css:177]: pending @PRABgit01

**Review by ankitadrixus08 — COMMENTED** [review]
please solve all comments and mark it as resolved. @PRABgit01

File: styled.css (hunk 22 above)
Comment (ankitadrixus08) [reply styled.css]: remove this container we need only one @PRABgit01

File: styled.css [hunk 28]
```diff
@@ -0,0 +1,299 @@
...
+  max-width: 1170px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.roadMapTitle {
+  margin: 64px;
+  color: #c6cdd9;
+  text-align: center;
+  font: sans-serif;
```
Comment (ankitadrixus08) [inline styled.css]: why sans sarif here @PRABgit01

File: styled.css [hunk 29]
```diff
@@ -0,0 +1,299 @@
...
+.swiperWrapper {
+  display: flex;
+  justify-content: center;
+  margin: 62px;
+}
+
+.swiperWrapperText {
+  max-width: 172px;
+  width: 100%;
+  font-size: 18px;
```
Comment (ankitadrixus08) [inline styled.css:103]: need line height here @PRABgit01

File: styled.css [hunk 30]
```diff
@@ -0,0 +1,299 @@
...
+  width: 16px;
+  height: 16px;
+  background-color: #fff;
+  border-radius: 25px;
+  top: -35px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+}
+
```
Comment (ankitadrixus08) [inline styled.css:164]: remove white space @PRABgit01

File: styled.css [hunk 31]
```diff
@@ -0,0 +1,299 @@
...
+  }
+
+  .leftDiv {
+    margin: 0 20px 0 0;
+    max-width: 130px;
+    width: 50%;
+    text-align: left;
+  }
+
+  .TitleMobile {
```
Comment (ankitadrixus08) [inline styled.css]: TitleMobile >> titleMobile

File: styled.css [hunk 32]
```diff
@@ -0,0 +1,299 @@
...
+
+  /* --second section ----- */
+  .swiperWrapper {
+    overflow-x: scroll;
+    justify-content: flex-start;
+    margin: 0px;
+  }
+
+  .swiperWrapperText {
+    min-width: 187px;
```
Comment (ankitadrixus08) [inline styled.css:234]: why you need this @PRABgit01

File: styled.css [hunk 33]
```diff
@@ -0,0 +1,299 @@
...
+  }
+
+  .roadmaproadmapDateMobile {
+    display: block;
+    max-width: 90px;
+    width: 100%;
+    background-color: #20e6f2;
+    padding: 8px 10px 6px;
+    margin: 0 0 15px;
+    box-shadow: inset 0 2px 2px #0000008a;
```
Comment (ankitadrixus08) [inline styled.css]: did you understand this code "#0000008a" can you please explain me @PRABgit01

File: styled.css [hunk 34]
```diff
@@ -0,0 +1,299 @@
...
+    margin: 0;
+    width: 8px;
+    height: 8px;
+    background-color: #fff;
+    border-radius: 25px;
+    border: none;
+    top: -15px;
+  }
+
+  .roadmaproadmapDateMobile {
```
Comment (ankitadrixus08) [inline styled.css]: roadmaproadmapDateMobile >> roadmapDate

here for mobile give different class mobileShow or mobileHide and use common for all @PRABgit01

File: styled.css [hunk 35]
```diff
@@ -0,0 +1,299 @@
...
+  }
+  .status {
+    top: 0px;
+  }
+  .statusWrapper {
+    padding: 10px 5px;
+  }
+
+  .roadMapTitle {
+    font-size: 20px;
```
Comment (ankitadrixus08) [inline styled.css:282]: need line height here @PRABgit01

File: styled.css [hunk 36]
```diff
@@ -0,0 +1,299 @@
...
+    margin: 45px;
+  }
+
+  .backgroundImagefeature {
+    max-width: 190px;
+    left: auto;
+    right: 0;
+    top: 249px;
+    transform: scaleX(-1);
```
Comment (ankitadrixus08) [inline styled.css:291]: why use scale here @PRABgit01

File: styled.css [hunk 37]
```diff
@@ -0,0 +1,299 @@
...
+  .statusWrapper {
+    padding: 10px 5px;
+  }
+
+  .roadMapTitle {
+    font-size: 20px;
+    margin: 45px;
+  }
+
+  .backgroundImagefeature {
```
Comment (ankitadrixus08) [inline styled.css]: backgroundImagefeature >> backgroundFeatureImage @PRABgit01

**Review by ankitadrixus08 — COMMENTED** [review]
too much repeated comments please check 2 times at your end 

18m

File: styled.css (hunk 26 above)
Comment (PRABgit01) [reply styled.css:177]: shield size was increasing automatically so used this i tryied in container but it was not working so use this

File: styled.css (hunk 32 above)
Comment (PRABgit01) [reply styled.css:234]: there was issue in my text size was increasing in small display

File: styled.css (hunk 36 above)
Comment (PRABgit01) [reply styled.css:291]: to flip the image

File: styled.css (hunk 25 above)
Comment (PRABgit01) [reply styled.css]: to move the image in top it was not adjusting its position

File: styled.css (hunk 33 above)
Comment (PRABgit01) [reply styled.css]: removed unwanted

File: Index.html [hunk 38]
```diff
@@ -0,0 +1,229 @@
...
+    </head>
+    <body>
+        <div class="wrapper">
+            <section class="bannerSection">
+                <div class="container">
+                    <h3 class="titleMobile">
+                        Join the Escrow Manager Program
+                    </h3>
+                    <div class="innerWrapper">
+
```
Comment (ankitadrixus08) [inline Index.html]: remove space @PRABgit01

File: Index.html [hunk 39]
```diff
@@ -0,0 +1,229 @@
...
+                        <div class="leftDiv">
+                            <div class="imageBox">
+                                <img class="imageDesign"
+                                    src="https://www.chainpals.io/assets/images/guest/png/become-escrow-manager.png" alt="become-escrow-manager">
```
Comment (ankitadrixus08) [inline Index.html]: become-escrow-manager >> become escrow manager @PRABgit01

File: Index.html [hunk 40]
```diff
@@ -0,0 +1,229 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <!-- --------roadmap section start---------- -->
+            <section class="roadMapSection">
+                <div class="roadMApContainer">
+                    <p class="roadMapTitle">ROAD MAP</p>
```
Comment (ankitadrixus08) [inline Index.html]: use heading tag here @PRABgit01

File: Index.html [hunk 41]
```diff
@@ -0,0 +1,229 @@
...
+                </div>
+                <div class="backgroundFeatureImage">
+                    <img
+                        src="https://www.chainpals.io/assets/images/guest/svgs/roadmap.svg" alt="shield Image"
```
Comment (ankitadrixus08) [inline Index.html]: shield Image > shield @PRABgit01

File: Index.html [hunk 42]
```diff
@@ -0,0 +1,229 @@
...
+                        class="shieldImage">
+                </div>
+                <div class="roadMapWrapper">
+                    <div class="swiperWrapper">
+                        <div class="swiperWrapperText">
+                            <div class="timeStampText">
+                                <span class="roadmapDate">Q4 2021</span>
+                            </div>
+                            <div class="statusWrapper">
+
```
Comment (ankitadrixus08) [inline Index.html]: remove white space @PRABgit01

File: Index.html [hunk 43]
```diff
@@ -0,0 +1,229 @@
...
+                                </div>
+                            </div>
+                        </div>
+                        <!-- -----------------code repeat for box -->
+                        <div class="swiperWrapperText">
+                            <div class="timeStampText">
+                                <span class="roadmapDate">Q1 2022</span>
+                            </div>
+                            <div class="statusWrapper">
+
```
Comment (ankitadrixus08) [inline Index.html]: same remove space @PRABgit01

File: styled.css [hunk 44]
```diff
@@ -0,0 +1,307 @@
+@font-face {
```
Comment (ankitadrixus08) [inline styled.css:1]: remove fonts/AvenirLTPro all fonts file as you are using only nexa here @PRABgit01

File: styled.css [hunk 45]
```diff
@@ -0,0 +1,307 @@
...
+}
+.imageDesign {
+  max-width: 342px;
+  width: 100%;
+}
+
+.description {
+  color: #fff;
+  font-size: 16px;
+  line-height: 4px;
```
Comment (ankitadrixus08) [inline styled.css]: remove duplicate code @PRABgit01

File: styled.css [hunk 46]
```diff
@@ -0,0 +1,307 @@
...
+  visibility: hidden;
+}
+
+/* -------CSS FOR DIFFERENT SECTION-------- */
+
+.inerWrapperRoadMap {
+  display: flex;
+}
+
+.roadMApContainer {
```
Comment (ankitadrixus08) [inline styled.css]: roadMApContainer >> roadMapContainer

File: Index.html (hunk 41 above)
Comment (PRABgit01) [reply Index.html]: ok

File: Index.html (hunk 40 above)
Comment (PRABgit01) [reply Index.html]: ok

File: Index.html (hunk 42 above)
Comment (PRABgit01) [reply Index.html]: done

File: styled.css (hunk 44 above)
Comment (PRABgit01) [reply styled.css:1]: ok ill fix it

File: styled.css (hunk 45 above)
Comment (PRABgit01) [reply styled.css]: mix bymistake

File: styled.css (hunk 46 above)
Comment (PRABgit01) [reply styled.css]: fixed it

**Review by ankitadrixus08 — APPROVED** [review]

**Pull request merged**